apps/channels/datamodels.py

```diff
@@ -1,14 +1,28 @@
 import phonenumbers
 from pydantic import BaseModel, Field, field_validator
+from typing_extensions import TypedDict
 
 from apps.channels.models import ChannelPlatform
 from apps.chat.channels import MESSAGE_TYPES
+from apps.files.models import File
+
+
+class Attachment(TypedDict):
+    file: File
+    type: str
+
+    class Config:
+        arbitrary_types_allowed = True
```

Comment:
Is this needed on a `TypedDict`? I thought that was a pydantic specific thing

Reply:
https://github.com/dimagi/open-chat-studio/pull/525/commits/dea2fd86a8737af55d61342c8a15d31b7573231a